pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -764,11 +766,12 @@ func newKubeVirtConfigForCR(cr *hcov1beta1.HyperConverged, namespace string) *co
 			Namespace: namespace,
 		},
 		// only virtconfig.SmbiosConfigKey, virtconfig.MachineTypeKey, virtconfig.SELinuxLauncherTypeKey and
-		// "debug.useEmulation" are going to be manipulated and only on HCO upgrades
+		// "debug.useEmulation" are going to be manipulated and only on HCO upgrades;
 		Data: map[string]string{
 			virtconfig.FeatureGatesKey:        "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar",
 			virtconfig.MigrationsConfigKey:    `{"nodeDrainTaintKey" : "node.kubernetes.io/unschedulable"}`,
 			virtconfig.SELinuxLauncherTypeKey: "virt_launcher.process",
+			virtconfig.NetworkInterfaceKey:    "masquerade",
```

Comment:
Don't you want to use the constant you defined in line 83?

Reply:
correct!
Done.